packages/expo-build-properties/src/android.ts

```diff
@@ -1,7 +1,12 @@
+import {
+  StringBoolean,
+  getMainApplicationOrThrow,
+} from '@expo/config-plugins/build/android/Manifest';
```

Comment:
```suggestion
```

to prevent further dependency issue, we should avoid use imports directly from `@expo/config-plugins`. using `expo/config-plugins` as much as we can. we can reuse imports as the following comments.

see more context from https://github.com/jakobo/expo-community-flipper/issues/16#issuecomment-1220254201